utility/importer/chrome_importer.cc

```diff
@@ -108,29 +111,43 @@ bool SetEncryptionKeyForPasswordImporting(
 }
 #endif
 
-class ScopedCopyFile {
- public:
-  explicit ScopedCopyFile(const base::FilePath& original_file_path) {
-    DCHECK(base::PathExists(original_file_path));
-    if (base::CreateTemporaryFile(&copied_file_path_))
-      copy_success_ = base::CopyFile(original_file_path, copied_file_path_);
-  }
-
-  ~ScopedCopyFile() {
-    if (base::PathExists(copied_file_path_))
-      base::DeleteFile(copied_file_path_, false);
-  }
+bool SetEncryptionKey(const base::FilePath& source_path) {
+#if defined(OS_LINUX)
+  // Set up crypt config.
+  std::unique_ptr<os_crypt::Config> config(new os_crypt::Config());
+  config->product_name = l10n_util::GetStringUTF8(IDS_PRODUCT_NAME);
+  config->should_use_preference = false;
+  config->user_data_path = source_path;
+  OSCrypt::SetConfig(std::move(config));
+  return true;
+#endif
 
-  bool copy_success() const { return copy_success_; }
-  ScopedCopyFile(const ScopedCopyFile&) = delete;
-  ScopedCopyFile& operator=(const ScopedCopyFile&) = delete;
+#if defined(OS_WIN)
+  base::FilePath local_state_path = source_path.DirName().Append(
+      base::FilePath::StringType(FILE_PATH_LITERAL("Local State")));
+  if (!base::PathExists(local_state_path))
+    return false;
+  if (!SetEncryptionKeyForPasswordImporting(local_state_path))
+    return false;
+  return true;
+#endif
 
-  base::FilePath copied_file_path() const { return copied_file_path_; }
+  return true;
+}
 
- private:
-  bool copy_success_ = false;
-  base::FilePath copied_file_path_;
-};
+base::string16 UnencryptedCardFromColumn(const sql::Statement& s,
```

Comment:
This should just be named `DecryptedCardFromColumn`

Reply:
Renamed.